worker/peergrouper/worker.go

```diff
@@ -749,7 +749,16 @@ func (w *pgWorker) getHASpaceFromConfig() (network.SpaceInfo, error) {
 		return network.SpaceInfo{}, errors.Trace(err)
 	}
 
-	space, err := w.config.State.Space(config.JujuHASpace())
+	// Before we changed the AlphaSpaceName to "alpha"
+	// instead of "", the call to SpaceByName would
+	// always work.  To mimic that behavior, if the
+	// JujuHASpace() is not set, use AlphaSpaceName
+	// instead.
+	jujuHASpace := config.JujuHASpace()
+	if jujuHASpace == "" {
+		jujuHASpace = network.AlphaSpaceName
```

Comment:
We will probably remove the scaffolding added earlier for passing a full space info to address filtering.
For now, we should return a zero-value `SpaceInfo` if the configuration is unset. Defaulting to the alpha space, then guarding against it in _desired.go_ means we would do no filtering even if an operator explicitly set it as the configured HA space.

Reply:
done